test/governance/extensions/GovernorTimelockAccess.test.js

```diff
@@ -112,6 +114,81 @@ contract('GovernorTimelockAccess', function (accounts) {
         expect(await this.mock.accessManager()).to.be.equal(this.manager.address);
       });
 
+      it('sets base delay (seconds)', async function () {
+        const baseDelay = time.duration.hours(10);
+
+        // Only through governance
+        await expectRevertCustomError(
+          this.mock.setBaseDelaySeconds(baseDelay, { from: voter1 }),
+          'GovernorOnlyExecutor',
+          [voter1],
+        );
+
+        await impersonate(this.mock.address);
+        const { receipt } = await this.mock.setBaseDelaySeconds(baseDelay, { from: this.mock.address });
+        expectEvent(receipt, 'BaseDelaySet', {
+          oldBaseDelaySeconds: '0',
+          newBaseDelaySeconds: baseDelay,
+        });
+
+        expect(await this.mock.baseDelaySeconds()).to.be.bignumber.eq(baseDelay);
+      });
+
+      it('sets access manager ignored', async function () {
+        const target = Wallet.generate().getChecksumAddressString();
```

Comment:
Why do we need to generate a wallet. IMO we could just use one of the accounts. 

On line 25 we can do 
```
const [admin, voter1, voter2, voter3, voter4, target] = accounts;
```

Reply:
Initially put it there with the intention of not mixing with the accounts, but makes sense. I named it `other` because it's not actually a target

Reply:
`other` is great